src/control/go.sum

```diff
@@ -523,6 +526,7 @@ google.golang.org/protobuf v1.25.0/go.mod h1:9JNX74DMeImyA3h4bdi1ymwjUzf21/xIlba
 google.golang.org/protobuf v1.26.0-rc.1/go.mod h1:jlhhOSvTdKEhbULTjvd4ARK9grFBp09yW+WbY/TyQbw=
 google.golang.org/protobuf v1.26.0 h1:bxAC2xTBsZGibn2RTntX0oH50xLsqy1OxA9tTL3p/lk=
 google.golang.org/protobuf v1.26.0/go.mod h1:9q0QmTI4eRPtz6boOQmLYwt+qCgq0jsYwAQnmE0givc=
+gopkg.in/alecthomas/kingpin.v2 v2.2.6 h1:jMFz6MfLP0/4fUyZle81rXUoxOBFi19VUFKVDOQfozc=
```

Comment:
What is bringing these new dependencies in? Generally speaking, we try to be conservative about adding new dependencies because it increases the release burden and size of the `src/control/vendor` directory.

Reply:
I am a complete new to GO, not sure how to check. This is what I got after compile.
Probably not really needed to be pushed but it is under 'src'.

Reply:
The tooling will update go.mod and go.sum if you import new packages somewhere. Probably you have these elsewhere in your project, as I don't see any evidence of their use in this PR. You can just revert the changes to those two files in your next commit.